builder/tart/step_disk_resize.go

```diff
@@ -26,6 +27,24 @@ func (s *stepResize) Run(ctx context.Context, state multistep.StateBag) multiste
 		return multistep.ActionContinue
 	}
 
+	// determine OS
```

Comment:
I wonder if it's easier to read `config.json` of the VM? What do you think? I don't have strong opinion on that.

Reply:
To me `uname(1)` approach looks way easier, plus we don't assume that Tart stores VMs at specific paths, in specific file names, with specific format.

Ideally we could just drop `step_disk_resize.go` and move all of the logic to `*.hcl`'s (as I've mentioned before in https://github.com/cirruslabs/packer-plugin-tart/pull/33#issuecomment-1300305178), but that's obviously out of scope for this PR.

Reply:
Let's stick with `uname` but the resizing logic IMO should live in the plugin. For example, users should not try to figure out which disk to resize. IMO that's too advanced.